src/fastfilter.h

```diff
@@ -29,12 +29,18 @@ constexpr bool isPow2(unsigned int num) { return num && !(num & (num - 1)); }
  * uint256 numbers that are sensitive to deliberately constructed collisions, be sure to keep NUM_HASH_FNS high enough
  * that the creation of collisions in the used bits is not feasible.
  *
+ * Note also that the input bits are used without obfuscation or mixing so if an attacker can control some input bits
+ * the attacker can cause collisions in some of the fast filter entries for his inputs.  This will cause higher false
+ * positive rates for these transactions.  For example, if the attacker can control 32 bits of the input, he can
+ * effectively reduce the number of hash functions in the fast filter by 2 because he has engineered a guaranteed
+ * collision for the two functions that use those bits.
+ *
```

Comment:
I've got those numbers in the paragraph above.  To expedite this, can you write whatever sentence works for you in a comment and I'll put it in if I agree with it?

Reply:
We need to turn around comment->change faster than once per day, especially when we are just dealing with comments -- and potentially short lived comment at that,  since they may be gone as soon as we find the time to invent or discover an efficient algorithm that obfuscates input bits.  Delaying this has already wasted @ptschip time because some collision reduction code in another PR was expecting the collision reduction in this one.  

Alternately,  lets get this approved and merged and we can open a new PR reworking the comments and any other small things.